eng/pipelines/coreclr/perf-non-wasm-jobs.yml

```diff
@@ -236,6 +236,39 @@ jobs:
         logicalmachine: 'perftiger'
         r2rRunType: --nor2r
 
+  # run coreclr perftiger microbenchmarks CrossBlockLocalAssertionProp perf jobs
+  - template: /eng/pipelines/common/platform-matrix.yml
+    parameters:
+      jobTemplate: /eng/pipelines/coreclr/templates/perf-job.yml
+      buildConfig: release
+      runtimeFlavor: coreclr
+      platforms:
+      - windows_x64
+      jobParameters:
+        testGroup: perf
+        liveLibrariesBuildConfig: Release
+        projectFile: microbenchmarks.proj
+        runKind: micro
+        runJobTemplate: /eng/pipelines/coreclr/templates/run-performance-job.yml
+        logicalmachine: 'perftiger'
+        r2rRunType: -ExperimentName crossblocklocalassertionprop
```

Comment:
I assume that this will change to the experiment name parameter once the other side is checked in?

Reply:
Damn copypasta. It should've been like that in the first place. Fixed now.